easyconfigs/v/VASPKIT/VASPKIT-1.5.1_fix-envvars.patch

```diff
@@ -0,0 +1,66 @@
++LDA_PATH                      =     <path/to/LDA/potential>        #  Path of LDA potential
++PBE_PATH                      =     <path/to/PBE/potential>        #  Path of PBE potential
++GGA_PATH                      =     <path/to/PW91/potential>       #  Path of PW91 potential
++VASPKIT_UTILITIES_PATH        =     $EBROOTVASPKIT/utilities       #  Path of VASPKIT
++PYTHON_BIN                    =     $EBROOTPYTHON/bin/python3      #  Python executable program with its installation path. Recommend Anaconda package 
+ POTCAR_TYPE                   =      PBE                           #  PBE, PW91 or LDA; Set PBE if you want to make PBE-POTCAR file
+ GW_POTCAR                     =     .FALSE.                        # .TRUE. or .FALSE.; For example, H_GW, O_GW will be chose when POTCAR_GW set to .TRUE.
+ RECOMMENDED_POTCAR            =     .TRUE.                         # .TRUE. or .FALSE.; The recommended PAW potential will be chose when RECOMMENDED_POTCAR set to .TRUE.
+@@ -42,7 +44,7 @@
+ INTERPOLATION_SPACING         =      0.04                          # Determines the number of interpolation grids, in unit of A in real-space or 1/A in reciprocal space (default: 0.04)
+ INTERPOLATION_METHOD          =     'cubic'                        # 'linear', 'cubic' (3rd order-spline interpolation), quartic (4th order-spline interpolation), or FFT available only for 2D and 3D grids (default method: 'cubic')
+ AUTO_SUBMIT_JOB               =     .FALSE.                        # .TRUE. or .FALSE. (default: .FASLE.). Whether to auto-submit vaspkit or vasp job or not.   
+-SUBMIT_JOB_COMMAND            =     'qsub job.sh'                  # The command line to submit job
++SUBMIT_JOB_COMMAND            =     'sbatch job.sh'                # The command line to submit job
+ AUTO_PLOT                     =     .FALSE.                        # TRUE. or .FALSE. (default: .FASLE.). Whether to auto-plot data graphs in the post-processing.
+ 
+ # New added in Version 1.4.1
+@@ -69,12 +71,14 @@
+ #| Must copy the block from #BEGIN_CUSTOMIZE_JOB_SCRIPT to #END_CUSTOMIZE_JOB_SCRIPT                                |             
+ #+------------------------------------------------------------------------------------------------------------------+
+ #BEGIN_CUSTOMIZE_JOB_SCRIPT   
+-#PBS -N name                  
+-#PBS -o out                   
+-#PBS -e err                   
+-#PBS -l nodes=2:ppn=4         
+-#PBS -r y                     
+-cd $PBS_O_WORKDIR             
++#!/bin/bash
++#SBATCH --job-name=name            # Job name
++#SBATCH --output=out               # Standard output
++#SBATCH --error=err                # Standard error
++#SBATCH --nodes=2                  # Number of nodes
++#SBATCH --ntasks-per-node=4        # Number of tasks per node
++#SBATCH --requeue                  # Requeue the job if it fails
```

Comment:
Not sure about these defaults – we probably don't want to split across two nodes for this small number of tasks unless there's a strong performance argument for doing so. Also, we'd probably want to include some dynamic variables in the `--output` and `--error` options, unless these files are being consumed elsewhere.